api/src/modules/analysis/providers/planning-units/adapters/are-puids-allowed-adapter.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@nestjs/common';
+import { ArePuidsAllowedPort } from '../are-puids-allowed.port';
+
+import { ScenariosPlanningUnitService } from '../../../../scenarios-planning-unit/scenarios-planning-unit.service';
+
+@Injectable()
+export class ArePuidsAllowedAdapter
+  extends ScenariosPlanningUnitService
+  implements ArePuidsAllowedPort {
+  async validate(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
just cosmetic - you can avoid this by prepending `_` to var names (at least with our eslint settings) - of course this means renaming the variable later on, so up to you - I don't have an IDE shortcut set up for this so I tend to prepend `_` 😄

Reply:
It is soon to be removed anyway (there https://github.com/Vizzuality/marxan-cloud/pull/165). As you said, this is a personal preference but leaving it silenced with `_prefix` leaves eye-state as "its all fine" while disable comment... stings the eyes.  I could as well just omit them but why? (: It does not matter much tho but if want to stick to project's style guide I am fine with that (as long as we note it down :)

Reply:
nah, i'd leave this to devs, either way it's clear